main.py

```diff
@@ -1,16 +1,36 @@
-# This is a sample Python script.
+import requests
+import numpy
+import pandas as pd
+import plotext as plt
 
-# Press Shift+F10 to execute it or replace it with your code.
-# Press Double Shift to search everywhere for classes, files, tool windows, actions, and settings.
+# requests library exercise
+url_posts = 'https://jsonplaceholder.typicode.com/posts'
+posts = requests.get(url_posts).json()
+url_users = 'https://jsonplaceholder.typicode.com/users'
+count = 0
+user_id = posts[0]['userId']
+for i in posts:
+    if i['userId'] == user_id:
+        count += 1
+# delete posts json array to save space
+del posts
+users = requests.get(url_users).json()
+print('Number of Posts by user: ' + users[user_id-1]['username'] + ' is ' + str(count))
 
+# numpy library exercise
+numbers = numpy.arange(10)
+# change the even values to 0
+numbers[numbers % 2 == 0] = 0
+print(numbers)
 
-def print_hi(name):
-    # Use a breakpoint in the code line below to debug your script.
-    print(f'Hi, {name}')  # Press Ctrl+F8 to toggle the breakpoint.
+# pandas exercise
+dates = pd.date_range("20130101", periods=6)
+df = pd.DataFrame(numpy.random.randn(6, 4), index=dates, columns=list("ABCD"))
+print(df)
 
-
-# Press the green button in the gutter to run the script.
-if __name__ == '__main__':
-    print_hi('PyCharm')
-
-# See PyCharm help at https://www.jetbrains.com/help/pycharm/
+# plotext exercise
+y = plt.sin(100, 3)
+plt.plot(y)
+plt.plotsize(100, 30)
+plt.title("Plot Example")
+plt.show()
```

Comment:
Same as the above comment. You can decide the file and function name for this. But this time create a class in the new file and add this function inside of the class and then use that in main.py.